data/mods/Medieval_Stuff/armor.json

```diff
@@ -0,0 +1,294 @@
+        "color" : "light_gray",
+        "covers" : ["TORSO", "ARMS"],
+        "to_hit" : -5,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "An ancient Roman laminar armor, made of overlapping metal strips connected to internal leather straps.",
+        "price" : 50000,
+        "material" : ["iron", "leather"],
+        "volume" : 24,
+        "cutting" : 0,
+        "warmth" : 15,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
+        "bashing" : 6,
+        "flags" : ["VARSIZE", "OUTER", "STURDY"],
+        "coverage" : 90,
+        "material_thickness" : 4
+    },{
+        "type" : "ARMOR",
+        "id" : "chainmail_hauberk",
+        "name" : "chainmail hauberk",
+        "weight" : 13977,
+        "color" : "light_red",
+        "covers" : ["TORSO", "ARMS", "LEGS"],
+        "to_hit" : -1,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A fully customized chainmail outfit, leaving the head uncovered.  The shirt, arms, and leggings have been modified with leather straps to deal with uneven weight distribution and to allow them to be used separately.",
+        "price" : 30000,
+        "material" : ["steel", "leather"],
+        "volume" : 10,
+        "cutting" : 0,
+        "warmth" : 0,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
+        "bashing" : -3,
+        "flags" : ["VARSIZE", "STURDY"],
+        "coverage" : 95,
+        "material_thickness" : 3
+    },{
+        "type" : "ARMOR",
+        "id" : "shendyt",
+        "name" : "light kilt",
+        "weight" : 1414,
+        "color" : "light_gray",
+        "covers" : ["LEGS"],
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A rather light garment typically made of linen or other plant fibers, designed for warm weather.",
+        "price" : 6500,
+        "material" : ["cotton", "null"],
+        "volume" : 6,
+        "cutting" : 0,
+        "warmth" : 5,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 0,
+        "bashing" : -5,
+        "flags" : ["VARSIZE"],
+        "coverage" : 50,
+        "material_thickness" : 2
+    },{
+        "type" : "ARMOR",
+        "id" : "legguard_bronze",
+        "name" : "pair of bronze greaves",
+        "name_plural" : "pairs of bronze greaves",
+        "category" : "armor",
+        "weight" : 2520,
+        "color" : "yellow",
+        "covers" : ["LEGS"],
+        "to_hit" : 1,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A pair of bronze leg guards with a simple leather lining.",
+        "flags" : ["BELTED", "STURDY"],
+        "price" : 40000,
+        "material" : ["bronze", "leather"],
+        "volume" : 22,
+        "cutting" : 0,
+        "warmth" : 15,
+        "phase" : "solid",
+        "environmental_protection" : 1,
```

Comment:
Environmental protection on something that gives 70% coverage and is made only to resist kinetic impact?